obspy/io/sac/pysac/header.py

```diff
@@ -0,0 +1,308 @@
+                  * IEQ2 (Felt earthquake)
+                  * IME (Marine explosion)
+                  * IEX (Other explosion)
+                  * INU (Nuclear explosion)
+                  * INC (Nuclear cavity collapse)
+                  * IO_ (Other source of known origin)
+                  * IR (Regional event of unknown origin)
+                  * IT (Teleseismic event of unknown origin)
+                  * IU (Undetermined or conflicting information)
+                  * IOTHER (Other)''',
+       'nevid': 'N    Event ID (CSS 3.0)',
+       'norid': 'N    Origin ID (CSS 3.0)',
+       'nwfid': 'N    Waveform ID (CSS 3.0)',
+       'khole': 'k    Hole identification if nuclear event.',
+       'dist': 'F    Station to event distance (km).',
+       'az': 'F    Event to station azimuth (degrees).',
+       'baz': 'F    Station to event azimuth (degrees).',
+       'gcarc': 'F    Station to event great circle arc length (degrees).',
+       'lcalda': 'L    TRUE if DIST AZ BAZ and GCARC are to be calculated '
+                 'from st event coordinates.',
+       'iqual': '''I    Quality of data [not currently used]:
+                  * IGOOD (Good data)
+                  * IGLCH (Glitches)
+                  * IDROP (Dropouts)
+                  * ILOWSN (Low signal to noise ratio)
+                  * IOTHER (Other)''',
+       'isynth': '''I    Synthetic data flag [not currently used]:
+                  * IRLDTA (Real data)
+                  * ????? (Flags for various synthetic seismogram codes)''',
+       'user0': 'F    User defined variable storage area 0.',
+       'user1': 'F    User defined variable storage area 1.',
+       'user2': 'F    User defined variable storage area 2.',
+       'user3': 'F    User defined variable storage area 3.',
+       'user4': 'F    User defined variable storage area 4.',
+       'user5': 'F    User defined variable storage area 5.',
+       'user6': 'F    User defined variable storage area 6.',
+       'user7': 'F    User defined variable storage area 7.',
+       'user8': 'F    User defined variable storage area 8.',
+       'user9': 'F    User defined variable storage area 9.',
+       'kuser0': 'K    User defined variable storage area 0.',
+       'kuser1': 'K    User defined variable storage area 1.',
+       'kuser2': 'K    User defined variable storage area 2.',
+       'lovrok': 'L    TRUE if it is okay to overwrite this file on disk.'}
+
+# because readable
+_hdritems = sorted(DOC.items())
+_hdrfmt = "{:10.10s} = {}"
+HDR_DESC = '\n'.join([_hdrfmt.format(_hdr, _doc) for _hdr, _doc in _hdritems])
+
+HEADER_DOCSTRING = """
+============ ==== ===============================================
+Field Name   Type Description
+============ ==== ===============================================
+""" + HDR_DESC + \
+        "\n============ ==== ==============================================="
+
+# Module documentation string
+__doc__ = MODULE_DOCSTRING + HEADER_DOCSTRING
```

Comment:
This should be used in the `__init__.py` to be better visible to the user.. such that it appears on the API front page for SAC.